docs/developers/tokens/zeta.mdx

```diff
@@ -0,0 +1,53 @@
+---
+sidebar_position: 1
+---
+
+import ZetaTokenTable from "@site/src/components/ZetaTokenTable/ZetaTokenTable";
+
+# ZETA
+
+## Native ZETA Token on ZetaChain
+
+The native token of the ZetaChain is called ZETA. It is a staking token, and is
+used to pay for transaction fees. ZetaChain node is built with Cosmos SDK
+framework and the ZETA token is implemented as a `sdk.Coin`. ZETA is the
+user-friendly symbol for the token, and the on-chain denom is `azeta`.
+
+1 ZETA = 10¹⁸ azeta.
+
+To query for account balance you can use the Cosmos HTTP API
+[`balances` endpoint](https://zetachain-athens.blockpi.network/lcd/v1/public/cosmos/bank/v1beta1/balances/zeta19nfaqu9wr0fktyyampva98ec025kjy0phww5um):
+
+```json
+{
+  "balances": [
+    {
+      "denom": "azeta",
+      "amount": "10000000000000000000"
+    }
+  ]
+}
+```
+
+To convert the value amount from `azeta` to ZETA, divide it by 10¹⁸. In the
+example above the balance is 10 ZETA.
+
+## Wrapped ZETA on ZetaChain
+
+ZETA can exists on ZetaChain in a wrapped form (WZETA) as a
```

Comment:
*can exist

Reply:
Also, I think it's worth explaining what wrapped means in this case.

Reply:
183c42b 68af21c

Reply:
I do need a good spellcheck for VS Code. You'd think with Copilot that would be part of the package 😁